zerver/openapi/zulip.yaml

```diff
@@ -282,6 +282,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonSuccess'
+  /attachments:
+    get:
+      description: Get all the attachments of the requesting user.
```

Comment:
We use `|` on the first line of all `description` fields now.

Reply:
(I'd encourage you to read the changes to this file; there's a lot of improvements to formatting we've done that you'll want to incorporate in this and your other PRs).